web/src/main/java/no/nav/ung/sak/web/app/tjenester/forvaltning/ForvaltningOppgaveRestTjeneste.java

```diff
@@ -41,19 +51,86 @@ public class ForvaltningOppgaveRestTjeneste {
+        }
+
+        final var fagsak = fagsakRepository.hentSakGittSaksnummer(saksnummer, false);
+
+        final var periode = finnPeriode(måned, fagsak);
+
+        final var startRapporteringTask = ProsessTaskData.forProsessTask(OpprettOppgaveForInntektsrapporteringTask.class);
+        startRapporteringTask.setAktørId(fagsak.get().getAktørId().getAktørId());
+        startRapporteringTask.setProperty(OpprettOppgaveForInntektsrapporteringTask.PERIODE_FOM, periode.getFomDato().format(DateTimeFormatter.ISO_LOCAL_DATE));
+        startRapporteringTask.setProperty(OpprettOppgaveForInntektsrapporteringTask.PERIODE_TOM, periode.getTomDato().format(DateTimeFormatter.ISO_LOCAL_DATE));
+        startRapporteringTask.setProperty(OpprettOppgaveForInntektsrapporteringTask.OPPGAVE_REF, UUID.randomUUID().toString());
+        prosessTaskTjeneste.lagre(startRapporteringTask);
+
+        return Response.ok().build();
+    }
+
+    private LocalDateInterval finnPeriode(MånedForRapportering måned, Optional<Fagsak> fagsak) {
+        final var sisteBehandling = behandlingRepository.hentSisteYtelsesBehandlingForFagsakId(fagsak.get().getId()).get();
+
+        final var periodisertMånedvis = månedsvisTidslinjeUtleder.periodiserMånedsvis(sisteBehandling.getId());
+
+        final var periode = periodisertMånedvis.stream()
+            // Enum ordinal er 0-indeksert og montvalue er 1-indeksert
+            .filter(it -> it.getFom().getMonthValue() == måned.ordinal() + 1)
+            .findFirst().map(LocalDateSegment::getLocalDateInterval).get();
+        return periode;
+    }
+
+    @POST
+    @Path("start-inntektskontroll")
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @Operation(description = "Starter inntektskontroll for sak", summary = ("Starter inntektskontroll for sak"), tags = "oppgave")
+    @BeskyttetRessurs(action = BeskyttetRessursActionAttributt.CREATE, resource = FAGSAK)
+    public Response startInntektskontroll(@NotNull @FormParam("saksnummer") @Parameter(description = "saksnummer", required = true) @Valid @TilpassetAbacAttributt(supplierClass = AbacAttributtSupplier.class) Saksnummer saksnummer,
+                                          @NotNull @QueryParam("måned") @Parameter(description = "måned", required = true) @Valid @TilpassetAbacAttributt(supplierClass = AbacAttributtEmptySupplier.class) MånedForRapportering måned) {
+        if (isProd) {
+            throw new IllegalArgumentException("Kan ikke kjøre denne tjenesten i prod");
+        }
+
+        final var fagsak = fagsakRepository.hentSakGittSaksnummer(saksnummer, false);
+        final var periode = finnPeriode(måned, fagsak);
+
+        final var startKontrollTask = ProsessTaskData.forProsessTask(OpprettRevurderingEllerOpprettDiffTask.class);
+        startKontrollTask.setFagsakId(fagsak.get().getId());
+        startKontrollTask.setProperty(BEHANDLING_ÅRSAK, BehandlingÅrsakType.RE_KONTROLL_REGISTER_INNTEKT.getKode());
```

Comment:
Er det riktig BehandlingÅrsakType for inntektsrapportering?

Reply:
Never mind. Ser det er et annet endepunkt.

Reply:
Dette er endepunkt for å starte inntektkontroll